mousetrap.js

```diff
@@ -1018,4 +1029,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
When you are using server side rendering. No `window` or `document` on the server side. It will throw an exception if we don't check that they are defined.